go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/Azure/azure-sdk-for-go/v54
+
+go 1.16
```

Comment:
This will prevent usage on older versions of Go.  For track 2 we've been using 1.13 as the min version.  Should do the same here.